scripts/apps/authoring/views/send-item.html

```diff
@@ -52,7 +52,9 @@
         </div>
 
         <div sd-toggle-box id="publishScheduleTimestamp" ng-if="mode === 'authoring' && showPublishSchedule()" data-title="{{ 'Publishing Schedule' | translate }}" data-open="true" data-style="dark">
-            <label class="label--lite" translate>Publish Schedule</label>
+            <label class="label--lite" translate>Publish Schedule
```

Comment:
now this button would be part of translate string and its title is not translated